tests/classification/test_accuracy.py

```diff
@@ -448,3 +448,9 @@ def test_negmetric_noneavg(noneavg=_negmetric_noneavg):
     assert torch.allclose(noneavg["res1"], result1, equal_nan=True)
     result2 = acc(noneavg["pred2"], noneavg["target2"])
     assert torch.allclose(noneavg["res2"], result2, equal_nan=True)
+
+
+def test_provide_superclass_kwargs():
```

Comment:
let's move this to something like general classification tests and parametrize over classes

Reply:
Moved it to the superclass test files, i.e., test_stat_scores and test_confusion_matrix. 
Any idea on how to make sure people build tests for this kind of situation going forward? Seems like a pretty easy thing to forget when adding new metrics classes or modifying the arguments of the current ones